src/components/PrimaryPanes/Outline.js

```diff
@@ -55,8 +54,7 @@ export class Outline extends Component<Props> {
         className="outline-list__element"
         onClick={() => this.selectItem(location)}
       >
-        <Svg name="function" />
-        <PreviewFunction func={{ name, parameterNames }} />
+        λ <PreviewFunction func={{ name, parameterNames }} />
```

Comment:
We do potentially want a way to style this element individually, so a tag around it would be nice.

Reply:
Will do. I think that might also fix the failing test.